docs/options api.md

```diff
@@ -13,14 +13,20 @@ need to create:
 - A new option class with a docstring detailing what the option will do to your user.
 - A `display_name` to be displayed on the webhost.
 - A new entry in the `option_definitions` dict for your World.
-By style and convention, the internal names should be snake_case. If the option supports having multiple sub_options
-such as Choice options, these can be defined with `option_my_sub_option`, where the preceding `option_` is required and
-stripped for users, so will show as `my_sub_option` in yaml files and if `auto_display_name` is True `My Sub Option`
-on the webhost. All options support `random` as a generic option. `random` chooses from any of the available
-values for that option, and is reserved by AP. You can set this as your default value but you cannot define your own
-new `option_random`.
+By style and convention, the internal names should be snake_case.
 
 ### Option Creation
+- If the option supports having multiple sub_options, such as Choice options, these can be defined with
```

Comment:
Isn't something like "distinct values" a better term than "sub options"? Not a native speaker, but as programmer sub-option sounds like a dict in a dict to me (i.e. `{"option": {"sub1": "val1", "sub2": "val2"}`).

Reply:
well in code we call them `options` and it creates a dict of them🤔

Reply:
it's only a dict if the player puts weights on it. the final .value is a distinct value. but yeah the word option is a bit ambiguous in general - my real problem is that `alias_my_sub_option_alt` is just very hard to follow, so i was hoping we might find a better word for it.